src/gui/optionsdialog.cpp

```diff
@@ -338,6 +338,7 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     connect(m_ui->checkAssociateMagnetLinks, &QAbstractButton::toggled, this, &ThisType::enableApplyButton);
     connect(m_ui->checkProgramUpdates, &QAbstractButton::toggled, this, &ThisType::enableApplyButton);
 #endif
+    connect(m_ui->checkBoxPerformanceWarnings, &QAbstractButton::toggled, this, &ThisType::enableApplyButton);
```

Comment:
Could you use either Warning or Warnings and not mix them?

Reply:
OK, fixed.